src/testclient/scripts/1.configureSpider/list-cloud.sh

```diff
@@ -8,23 +8,42 @@
     echo "## 0. List Cloud Connction Config(s)"
     echo "####################################################################"
 
-    # for Cloud Connection Config Info
-    curl -H "${AUTH}" -sX GET http://$SpiderServer/spider/connectionconfig | jq ''
-    echo ""
-
-
     # for Cloud Region Info
-    curl -H "${AUTH}" -sX GET http://$SpiderServer/spider/region | jq ''
+    echo "[Cloud Region]"
+    curl -H "${AUTH}" -sX GET http://$SpiderServer/spider/region |
+    jq '.region' |
+    jq -r '(["RegionName","ProviderName","Region","Zone"] | (., map(length*"-"))), (.[] | [.RegionName, .ProviderName]) | @tsv' |
```

Comment:
```suggestion
    jq -r '(["RegionName","ProviderName","Region","Zone"] | (., map(length*"-"))), (.[] | [.RegionName, .ProviderName, .KeyValueInfoList.[0].Value, .KeyValueInfoList.[1].Value]) | @tsv' |
```

Reply:
이렇게 하니까 되네요 😊

```suggestion
    jq -r '(["RegionName","ProviderName","Region","Zone"] | (., map(length*"-"))), (.[] | [.RegionName, .ProviderName, .KeyValueInfoList[0].Value, .KeyValueInfoList[1].Value]) | @tsv' |
```

그런데.. `KeyValueInfoList` 의 내용이 웬만해서는 `Region`, `Zone` 순서대로 오는데,
JSON body field들의 순서가 보장되지는 않으므로
Region과 Zone이 반대로 출력되는 경우도 있을 수 있겠습니다. 😢

Reply:
네 일단 큰 문제는 없어보이니! 업데이트해주신 현재 내용으로 머지하겠습니다~~~ 
감사감사!★